highfidelity/hifi
# Create - native desktop windows

## Native desktop windows
This PR separates the Create app interface for Desktop in two system-level windows. The Entity List and the Create Tools. The `OverlaySystemWindow` class has been added to the JS API to allow the creation of these system-level windows. 

### QA test plan
1. Start interface in desktop mode
2. Open the create app  ( two windows should appear.  Entity List - left side of the window,  Create Tools - right side of the window)
3. Move the windows around, and resize them. (remember their positions)
4. Switch to VR mode. (The create app should de-activate.)
5. Open the create app. (The create app should function as it does in master, all the create tabs available in tablet   (create, entity list, properties, grid, particle) )
6. Switch back to desktop mode.
7. Open the create app. ( the two windows should appear in their last known locations)
8. Close both desktop windows (Entity List and Create Tools) with the [x].
9. De-activate the create app.
10. Activate the create app. ( the two windows should appear in their last known locations)
11. Smoke test create functionality. (e.g. Create Models, Particles and Shapes)

[edit, jcII: test on win/desk, mac/desk, win/rift, and win/vive. Perform minimum 30 minutes exploratory on each platform after test plan. I'm estimating 90 minutes per platform, 6 hours total. Please tell me if that's too low.]
